cmd/cluster/resize/infra_node.go

```diff
@@ -27,11 +26,12 @@ import (
 )
 
 const (
-	twentyMinuteTimeout                = 20 * time.Minute
-	twentySecondIncrement              = 20 * time.Second
-	resizedInfraNodeServiceLogTemplate = "https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/infranode_resized_auto.json"
-	infraNodeLabel                     = "node-role.kubernetes.io/infra"
-	temporaryInfraNodeLabel            = "osdctl.openshift.io/infra-resize-temporary-machinepool"
+	twentyMinuteTimeout                   = 20 * time.Minute
+	twentySecondIncrement                 = 20 * time.Second
+	resizedInfraNodeServiceLogTemplate    = "https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/infranode_resized_auto.json"
+	GCPresizedInfraNodeServiceLogTemplate = "https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/gcp/GCP_infranode_resized_auto.json"
```

Comment:
I don't think the service log listed here as "GCP" is any better - the documentation it links to still only talks about ROSA

Reply:
@mjlshen GCP SL points to OSD documentation not the ROSA documentation. We do not have a separate documentation for GCP on OSD.  AWS on OSD and GCP on OSD are in single documentation.